webtester-core/src/test/java/integration/AbstractWebTesterIntegrationTest.java

```diff
@@ -54,6 +55,10 @@ protected String getHTMLFilePath() {
 
     // UTILITY
 
+    protected void assertPageObjectCanBeInitialized(PageObject object) {
```

Comment:
I need this assertion for other PageObjects which are based on GenericTextField. 
That is why I added it on this branch, so I dont have to add it on all the other Branches.